ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -1177,8 +1204,12 @@ private void verifyPutFailure(MockId idToPut, StoreErrorCodes expectedErrorCode)
    * @param expectedErrorCode the expected {@link StoreErrorCodes} for the failure.
    */
   private void verifyDeleteFailure(MockId idToDelete, StoreErrorCodes expectedErrorCode) {
-    MessageInfo info = new MessageInfo(idToDelete, DELETE_RECORD_SIZE, Utils.getRandomShort(TestUtils.RANDOM),
-        Utils.getRandomShort(TestUtils.RANDOM), System.currentTimeMillis());
+    short accoundId = accountIdMap.containsKey(idToDelete) == true ? accountIdMap.get(idToDelete)
```

Comment:
didn't understand this - why do this?

Reply:
The reason to create hashMap is we want to match accountId and containerId. Previously test code always use random.

However, some test gives unknown blobId to test ID_Not_Found behavior.
For example, verifyDeleteFailure(getUniqueId(), StoreErrorCodes.ID_Not_Found);
ID generated by getUniqueId() doesn't exist in HashMap. null returned by hashMap is not compatible with short, which required by MessageInfo().